src/services/projectsUsers/projectsUsersTypes.ts

```diff
@@ -0,0 +1,10 @@
+export interface ProjectsUsersResponse {
+  data: {
+    users: {
```

Comment:
There is no 'users' property within 'data', it holds the array of user objects directly.